src/hardware/input/mouse_config.cpp

```diff
@@ -127,56 +116,132 @@ bool MouseConfig::ParseCOMModel(const std::string_view model_str,
+	} else if (capture_str == capture_type_onstart_str) {
+		mouse_config.capture = MouseCapture::OnStart;
+	} else if (capture_str == capture_type_nomouse_str) {
+		mouse_config.capture = MouseCapture::NoMouse;
+	} else {
+		assert(false);
+	}
 }
 
-bool MouseConfig::ParsePS2Model(const std::string_view model_str, MouseModelPS2& model)
+static void SetPS2Model(const std::string_view model_str)
 {
 	if (model_str == model_ps2_standard_str) {
-		model = MouseModelPS2::Standard;
+		mouse_config.model_ps2 = MouseModelPS2::Standard;
 	} else if (model_str == model_ps2_intellimouse_str) {
-		model = MouseModelPS2::IntelliMouse;
+		mouse_config.model_ps2 = MouseModelPS2::IntelliMouse;
 	} else if (model_str == model_ps2_explorer_str) {
-		model = MouseModelPS2::Explorer;
-	} else if (auto as_bool = parse_bool_setting(model_str);
-	           as_bool && *as_bool == false) {
-		model = MouseModelPS2::NoMouse;
+		mouse_config.model_ps2 = MouseModelPS2::Explorer;
+	} else if (model_str == model_ps2_nomouse_str) {
+		mouse_config.model_ps2 = MouseModelPS2::NoMouse;
 	} else {
-		return false;
+		assert(false);
 	}
-	return true;
 }
 
-const std::vector<uint16_t> &MouseConfig::GetValidMinRateList()
+static void SetCOMModel(const std::string_view model_str)
 {
-	return list_rates;
+	[[maybe_unused]] const auto result = MouseConfig::ParseCOMModel(
+	        model_str, mouse_config.model_com, mouse_config.model_com_auto_msm);
+	assert(result);
+}
+
+static void SetSensitivity(const std::string_view sensitivity_str)
+{
+	// Coefficient to convert percentage in integer to float
+	constexpr float coeff = 0.01f;
+
+	// Split input string into values
+	auto values_str = split(sensitivity_str, " \t,;");
+	if (values_str.size() > 2) {
+		LOG_WARNING("CONFIG: Too many values in 'mouse_sensitivity', using defaults");
+		return;
+	}
+
+	// If no values given, use defaults
+	if (values_str.empty()) {
+		const auto &user_default = mouse_predefined.sensitivity_user_default;
+		const auto default_value = coeff * user_default;
+		mouse_config.sensitivity_coeff_x = default_value;
+		mouse_config.sensitivity_coeff_y = default_value;
+		return;
+	}
+
+	// Convert values to integers
+	std::vector<int> values_int = {};
+
+	bool out_of_range   = false;
+	const int value_min = -mouse_predefined.sensitivity_user_max;
+	const int value_max = mouse_predefined.sensitivity_user_max;
+	for (auto& value_str : values_str) {
+		// Remove trailing '%' signs, if present
+		if (ends_with(value_str, "%")) {
+			value_str.pop_back();
+		}
+
+		const auto value = parse_int(value_str);
+		if (!value) {
+			LOG_WARNING("CONFIG: Syntax error in 'mouse_sensitivity', using defaults");
+			return;
+		}
+
+		const auto value_clamped = clamp(*value, value_min, value_max);
+		if (value_clamped != *value) {
+			out_of_range = true;
+		}
+		values_int.push_back(value_clamped);
+	}
+
+	if (out_of_range) {
+		LOG_WARNING("CONFIG: 'mouse_sensitivity' adjusted to range %+d - %+d",
```

Comment:
Let's use the `MOUSE: ` prefix in all warnings.

Reply:
OK, changed.